src/app/character-tile/CharacterTile.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import clsx from 'clsx';
+import { DimStore } from '../inventory/store-types';
+import { AppIcon, powerActionIcon } from '../shell/icons';
+import './CharacterTile.scss';
+
+const CharacterEmblem = ({ store }: { store: DimStore }) => (
+  <div
+    className={clsx('emblem', { vault: store.isVault })}
+    style={{ backgroundImage: `url("${store.icon}")` }}
+  />
+);
+
+export const CharacterTile = ({ store }: { store: DimStore }) => (
```

Comment:
It'd be great to get some light documentation that explains what each component is used for